js/views/device_pairing_dialog_view.js

```diff
@@ -37,10 +48,12 @@
         requestAcceptedTitle: i18n('devicePairingAccepted'),
         startPairingText: i18n('pairNewDevice'),
         cancelText: i18n('cancel'),
+        UnpairDevice: i18n('unpairDevice'),
```

Comment:
```suggestion
        unpairDevice: i18n('unpairDevice'),
```
This unsettles me haha, also corresponding usage above

Reply:
Good catch :)